### ISSUE TITLE
Pathfinder loading issue (on game start)

### ISSUE THREAD

**Issue by CityScrambler** [KSPModdingLibs/KSPCommunityFixes#134]
1.12.5.3190 (WindowsPlayer x64)

Specific Error message that popped up ingame:
```

Loading has failed due to an unhandled error

Failure in subsystem : Part compilation
Part: Sombrero Solar Array (WildBlueIndustries/Pathfinder/Parts/Utility/Sombrero/WBI_Sombrero)

System.NullReferenceException: Object reference not set to an instance of an object
  at ModuleDeployablePart.AssumeDragCubePosition (System.String name) [0x00088] in <4b449f2841f84227adfaad3149c8fdba>:0 
  at (wrapper dynamic-method) DragCubeSystem+<RenderDragCubes>d__34.DragCubeSystem+<RenderDragCubes>d__34.MoveNext_Patch0(DragCubeSystem/<RenderDragCubes>d__34)
  at KSPCommunityFixes.Performance.KSPCFFastLoader+<FrameUnlockedCoroutine>d__61.MoveNext () [0x00118] in <bcbfb6a72174435890d38ec84787b02a>:0 


```

`[KSP.log](https://github.com/KSPModdingLibs/KSPCommunityFixes/files/11104298/KSP.log)`

**Comment by gotmachine** [comment]
Yeah, this is caused by the fix for https://github.com/KSPModdingLibs/KSPCommunityFixes/issues/133 to be overly conservative with errors that aren't actually fatal in stock. Will push a fixed version ASAP, in the meantime, you can downgrade to KSPCF 1.25.3 to avoid the issue.

**Comment by CityScrambler** [comment]
> Yeah, this is caused by the fix for #133 to be overly conservative with errors that aren't actually fatal in stock. Will push a fixed version ASAP, in the meantime, you can downgrade to KSPCF 1.25.3 to avoid the issue.

Great. Works with KSPCF 1.25.3 indeed. Thanks
